### PR TITLE
fix/capitalization changed capitals

### PR DESCRIPTION
**Pull request by christophervelo**
**I changed some of the capitals, please forgive me senpai**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -18,8 +18,8 @@
   <div>
     <h1>This is Chris's Portfolio</h1>
     <a href="about.html" class="underline-on-hover">About Me</a>
-    <h2>About all my work</h2>
-    <h3>Cindy was here writing code</h3>
+    <h2>about all my work</h2>
```
Comment (carrillocc) [inline index.html:21]: Chris, this needs to be in all caps, but great attempt!

File: index.html [hunk 2]
```diff
@@ -18,8 +18,8 @@
...
+    <h3>cindy was here writing code</h3>
```
Comment (carrillocc) [inline index.html:22]: You should add an explanation point at the end of the sentence! :D

File: index.html [hunk 3]
```diff
@@ -18,8 +18,8 @@
...
     <ol>
       <li>Islamorada</li>
```
Comment (carrillocc) [inline index.html:24]: Great work here!

**Review by carrillocc — CHANGES_REQUESTED** [review]
Just a couple changes, thanks!

File: index.html (hunk 1 above)
Comment (christophervelo) [reply index.html:21]: Did you just want the about in all caps or the entire sentence?

Comment (carrillocc) [reply index.html:21]: My bad, should have been more clear, just the `About` word

File: index.html [hunk 4]
```diff
@@ -18,8 +18,8 @@
   <div>
     <h1>This is Chris's Portfolio</h1>
```
Comment (carrillocc) [inline index.html:19]: I know you did not touch this line, but can you add a period at the end of this sentence!

**Review by carrillocc — CHANGES_REQUESTED** [review]
one more comment!

**Review by carrillocc — APPROVED** [review]
Great work Chris!

**Pull request merged**